### DISCUSSION TITLE
Change availability per device?

### DISCUSSION THREAD

**Discussion by realjax** [Koenkk/zigbee2mqtt#19409] [General]
I know I can set the availability check for all devices through the frontend. But I cant seem to find the setting on a per device level. Is this only possible by editing the yaml?

**Comment by realjax** [comment]
so I assume this is not possible then. Might as well close this.

**Reply by francisp2** [reply]
Maybe ask in the frontend repo

https://github.com/nurikk/zigbee2mqtt-frontend
